E2a-TRAS/CL01/README.md

```diff
@@ -0,0 +1,5 @@
+# Test instructions
+
+| Documentation Code   | Test Title                    | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|----------------------|-------------------------------|---------------|-----------| ----------------|------------|---------|------------|
+| IFC4.3AbRV_E2a_CL01  | (RFI) Classify objects        | E2a (TRAS)    | CL01      | Evandro Alfieri | RFI        | 1.0     | DD.MM.YYYY |
```

Comment:
Simile.

Reply:
Made the [PR](https://github.com/bSI-InfraRoom/MVD-Infra-Test-Instructions/pull/43) this morning. Just added you as reviewer